index-handlers/src/test/java/no/sikt/nva/nvi/index/aws/OpenSearchClientTest.java

```diff
@@ -575,7 +577,25 @@ void organizationAggregationShouldNotContainAggregationsForOtherTopLevelOrgs()
         assertThat(orgIds, not(containsInAnyOrder(NTNU_INSTITUTION_ID.toString())));
     }
 
-    @NotNull
+    @Test
+    void shouldExcludeFields() throws IOException, InterruptedException {
+        var document = documentWithContributors();
+        addDocumentsToIndex(document);
+        var searchParameters = defaultSearchParameters()
+                                   .withExcludeFields(List.of("publicationDetails.contributors"))
```

Comment:
Is this behavior intended to be the default for all queries, or only for queries that explicitly exclude contributors? If it's the default, perhaps add `.withExcludeFields(List.of("publicationDetails.contributors"))` directly to the helper method `defaultSearchParameters`?

Reply:
It's not intended to be the default, `publicationDetails.contributors` is still relevant for nvi search api